pallets/elections/src/impls.rs

```diff
@@ -208,12 +215,12 @@ where
     }
 
     fn adjust_rewards_for_session() {
-        let active_era = match T::EraInfoProvider::active_era() {
-            Some(ae) if ae > 0 => ae,
+        match T::EraInfoProvider::active_era() {
```

Comment:
`if T::EraInfoProvider::active_era().unwrap_or(0) == 0 { return }`
or 
`if T::EraInfoProvider::active_era().filter(|e| e>0).is_some() == 0 { return }`

Reply:
first one is the winner